src/language-service/src/home-assistant/haConnection.ts

```diff
@@ -130,9 +130,11 @@ export class HaConnection implements IHaConnection {
           return reject();
         }
         ha.subscribeEntities(this.connection, (entities) => {
-          console.log(
-            `Got ${Object.keys(entities).length} entities from Home Assistant`
-          );
+          if (shouldLog) {
```

Comment:
Revert that when a simple axios GET call outside language service is implemented.